rust/processor/src/schema.rs

```diff
@@ -518,6 +539,80 @@ diesel::table! {
     }
 }
 
+diesel::table! {
+    fungible_asset_activities (transaction_version, event_index) {
+        transaction_version -> Int8,
+        event_index -> Int8,
+        #[max_length = 66]
+        owner_address -> Varchar,
+        #[max_length = 66]
+        storage_id -> Varchar,
```

Comment:
What does this field store?

Reply:
Same thing as storage_id in other tables.